index.html

```diff
@@ -185,9 +185,76 @@ <h2>Skills</h2>
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
-      <form action="#"></form>
+      <form action="https://formspree.io/f/xknagban" method="post">
+        <fieldset>
+          <input
+            type="text"
+            name="full-name"
+            placeholder="Full name"
+            required
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="email-address"
+            placeholder="Email Address"
+            class="name"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter a Message"
+            id=""
+            cols="30"
+            rows="10"
+            required
+            maxlength="500"
+          ></textarea>
+        </fieldset>
+        <button class="btn" type="submit">Get in touch</button>
+      </form>
     </section>
 
+    <footer class="footer">
+      <div class="social-icons">
+        <ul>
+          <li>
+            <a href="#"
```

Comment:
- [Optional] There seems to be a  light adjustment needed between your `button` and the social links. I've attached the visual view for you to assess alongside th figma deign. I strongly suggest you make the change. 👍 

# Your design :
![image](https://user-images.githubusercontent.com/50990925/213091376-73730792-f066-4bcb-a5c8-d9d134f831da.png)

#Figma Design:
![image](https://user-images.githubusercontent.com/50990925/213091494-e7b2ef7d-dcb4-44ab-b803-8cbcc8c37843.png)